src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -8,13 +8,16 @@ export function RadioGroup<TKeys extends FieldValues>(
   props: RadioGroupProps & {
     control: Control;
     name: keyof TKeys;
+    // eslint-disable-next-line react/require-default-props
+    controllerProps?: Pick<ControllerProps<typeof $RadioGroup, FieldValues>, 'rules' | 'defaultValue'>;
```

Comment:
@jaymehta1986 instead of adding single Controller fields, I thought it'd be more future-proof to add them in a single `controllerProps` object to pass to the react-hook-form `Controller`.